examples/adjtime/adjtime_main.c

```diff
@@ -148,7 +148,7 @@ int main(int argc, FAR char *argv[])
   parse_args(&delta, argc, argv);
 
   printf("Delta time is %ld seconds and %ld micro seconds.\n",
-         delta.tv_sec, delta.tv_usec);
+         (long)delta.tv_sec, delta.tv_usec);
```

Comment:
Maybe better cast to `intmax_t` and use `PRIdMAX` instead?

Reply:
Or choose printf specifier based of `CONFIG_SYSTEM_TIME64`

Reply:
Technically yeah, but considering this is a delta time, it seems unlikely anyone would want to use the adjtime example program to change time by 70 years with rate adjustment.

The current implementation of `parse_args()` in the example limits values to `unsigned long`. I think that is a bit questionable, as the adjustment delta can be negative also. It does work with negative values currently, but that is probably more due to luck than design.

In any case, I consider the adjtime example quite low importance and I'm not interested in spending much time on it, just thought to fix a bug when I hit it.

Reply:
Ok. Let's keep it